gromacs/tools.py

```diff
@@ -94,177 +108,244 @@ def _generate_sphinx_class_string(clsname):
     'mdrun': 'mdrun',
     'make_ndx': 'make_ndx',
     'make_edi': 'make_edi',
-    'dump': 'gmxdump',
-    'check': 'gmxcheck',
     'genrestr': 'genrestr',
     'genion': 'genion',
     'genconf': 'genconf',
     'do_dssp': 'do_dssp',
+
+    # changed names
+    'convert_tpr': 'tpbconv',
+    'dump': 'gmxdump',
+    'check': 'gmxcheck',
     'solvate': 'genbox',
+    'distance': 'g_dist',
+    'sasa': 'g_sas',
+    'gangle': 'g_sgangle',
+
+    # 5.0.5 compatibility hack
+    'tpbconv': 'convert_tpr'
```

Comment:
I think I mentioned previously something along the lines "oh, this line could be merged with the aliases". But when looking at it in context, this seems dumb because we already have
```python
 'convert_tpr': 'tpbconv',
```

I am not really sure what's even happening now – does `convert_tpr` disappear?

I *think* we can just completely remove the line. The original motivation was that 5.0.x still shipped with `g_*` tools but they were inconsistent and did not include `tpbconv` anymore but had rather renamed it to `convert_tpr`. The hack predated the aliasing system. 

Let's remove the line and if there are any bug reports, we will address them...

Sorry about that.